test/features/currency/currency_dropdown_test.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:didpay/features/currency/currency_dropdown.dart';
+import 'package:didpay/features/payment/payment_amount_state.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+  
+import '../../helpers/test_data.dart';
+import '../../helpers/widget_helpers.dart';
+
+void main() async {
+  await TestData.initializeDids();
+
+  late ValueNotifier<PaymentAmountState?> mockState;
+  const testCurrency = 'USD';
+
+  setUp(() {
+    // Initialize mockState with a non-null offeringsMap
+    mockState = ValueNotifier<PaymentAmountState?>(
+      PaymentAmountState(
+        offeringsMap: TestData.getOfferingsMap(),
+      ), // Use TestData as suggested
+    );
+  });
+
+  Widget createWidgetUnderTest() {
+    return WidgetHelpers.testableWidget(
+      // Use WidgetHelpers for consistent test setup
+      child: CurrencyDropdown(
+        paymentCurrency: testCurrency,
+        state: mockState,
+      ),
+    );
+  }
```

Comment:
try to follow the existing test suite in terms of naming/initializing this widget (e.g. see [payout_test.dart](https://github.com/TBD54566975/didpay/blob/824cc4cfb9d67f71eec31bf50fee8474a84bb82a/test/features/payout/payout_test.dart#L15-L26))